src/wavesurfer.js

```diff
@@ -1168,6 +1173,24 @@ export default class WaveSurfer extends util.Observer {
         this.drawBuffer();
     }
 
+    /**
+     * Hide channels from being drawn on the waveform if splitting channels.
+     * 
+     * For example, if we want to draw only the peaks for the right stereo channel:
+     *
```

Comment:
last nit: can you use an `@example` here, see https://jsdoc.app/tags-example.html